StoreTestWPF.ViewModel/ViewModels/ViewModelBase.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace StoreTestWPF.ViewModel.ViewModels
+{
+    public class ViewModelBase : INotifyPropertyChanged
+    {
+        public event PropertyChangedEventHandler PropertyChanged;
+
+        public void OnPropertyChanged([CallerMemberName] string prop = "")
+        {
+            if (PropertyChanged != null)
+                PropertyChanged(this, new PropertyChangedEventArgs(prop));
```

Comment:
Missed 'this.'

Reply:
We can use null-propogation operator:
this.PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(prop));

Reply:
Fixed